Model/Sales/Repository/PackageRepository.php

```diff
@@ -64,6 +69,20 @@ public function setWeightFromQuoteProducts($products)
         return $this;
     }
 
+    /**
+     * @param array $products
+     *
+     * @return PackageRepository
+     */
+    public function productWithoutDeliveryOptions(array $products)
```

Comment:
Volgens mij moet je ook dit kunnen doen:

```suggestion
    public function productWithoutDeliveryOptions(array $products): PackageRepository
```
Als je dit veranderd dan moet je even kijken of het nog werkt

Reply:
je kan ook 
```suggestion
    public function productWithoutDeliveryOptions(array $products): self
```
doen

Reply:
done